packages/api-server/api_server/routes/beacons.py

```diff
@@ -0,0 +1,54 @@
+from typing import List
+
+from fastapi import HTTPException
+from rx import operators as rxops
+
+from api_server.fast_io import FastIORouter, SubscriptionRequest
+from api_server.models import tortoise_models as ttm
+from api_server.rmf_io import beacon_events
+
+router = FastIORouter(tags=["Beacons"])
+
+
+@router.sub("", response_model=ttm.BeaconStatePydantic)
+async def sub_beacons(_req: SubscriptionRequest):
+    return beacon_events.beacons.pipe(rxops.filter(lambda x: x is not None))
+
+
+@router.get("", response_model=List[ttm.BeaconStatePydantic])
+async def get_beacons():
+    beacons = await ttm.BeaconState.all()
+    return [await ttm.BeaconStatePydantic.from_tortoise_orm(a) for a in beacons]
+
+
+@router.get("/{beacon_id}", response_model=ttm.BeaconStatePydantic)
+async def get_beacon(beacon_id: str):
+    beacon_state = await ttm.BeaconState.get_or_none(id=beacon_id)
+    if beacon_state is None:
+        raise HTTPException(404, f"Beacon with ID {id} not found")
```

Comment:
```suggestion
        raise HTTPException(404, f"Beacon with ID {beacon_id} not found")
```

Otherwise I am getting the following error if I submit a non-existent beacon ID:
```json
{
  "detail": "Beacon with ID <built-in function id> not found"
}
```

Reply:
done! https://github.com/open-rmf/rmf-web/pull/726/commits/9bc51121dae9622d55f4267cefd4f0969af92385, thanks for catching that